From 1670cc88654c3f091a221cce5899a0df7ed4a123 Mon Sep 17 00:00:00 2001
From: Christian Bruel <christian.bruel@foss.st.com>
Date: Mon, 11 Dec 2023 14:30:43 +0100
Subject: [PATCH] arm64: dts: st: add sleep pins for pcie in stm32mp257f-ev1

Add analog gpio pin configuration ("sleep") to manage power mode on
stm32mp25.

Signed-off-by: Christian Bruel <christian.bruel@foss.st.com>
Change-Id: I36628562e5f2b1d5ccf8a6a5b4469c41cc0ce0b7
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -544,9 +544,10 @@
 };
 
 &pcie_rc {
-	pinctrl-names = "default", "init";
+	pinctrl-names = "default", "init", "sleep";
 	pinctrl-0 = <&pcie_pins_a>;
 	pinctrl-1 = <&pcie_init_pins_a>;
+	pinctrl-2 = <&pcie_sleep_pins_a>;
 	reset-gpios = <&gpioj 8 GPIO_ACTIVE_LOW>;
 	status = "okay";
 };
